### PR TITLE
#7 fix warning when playing audio

### LINKED ISSUE #7: Warning message when playing audio
# Problem description 

The following warning is outputted to the console when the bot plays audio : 

> Guessed Channel Layout for Input Stream #0.0 : mono

Looks like it has no effect but it would be nice to see if it can be supressed or if we can play audio with a parameter than prevents the message from showing up.

### PR DESCRIPTION
**Pull request by benduhoux**
# Reason for PR 

This fixes #7 .

# Change description

The FFmpegPCMAudio call was not using any options, making ffmpeg guess the channel layout (mono or stereo I suppose) every time it created a media file.
I added the option to specifiy the channel layout in the call. Because ffmpeg always guessed mono before and it worked, I just chose mono.

### CODE REVIEW

File: dj_mgr.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import discord
 from googleapiclient.discovery import build
-import vlc
```
Comment (benduhoux) [inline dj_mgr.py:3]: PArt of the module clean-up, vlc is not used.

File: dj_mgr.py [hunk 2]
```diff
@@ -18,14 +17,14 @@ def __init__(self, sazerac):
     def is_playing(self):
         try:
             return self.sazerac.client.voice_clients[0].is_playing()
-        except IndexError as e:
```
Comment (benduhoux) [inline dj_mgr.py:21]: Removing a couple of unused vairables

File: dj_mgr.py [hunk 3]
```diff
@@ -119,9 +119,3 @@ def find_song(self, url):
         best_url = pafy_video.getbestaudio().url
 
         return best_url, yt_title
-
```
Comment (benduhoux) [inline dj_mgr.py:122]: Cleaning up extra new lines, removing warnings

File: dj_mgr.py [hunk 4]
```diff
@@ -109,7 +109,7 @@ def find_song(self, url):
         self.sazerac.logger.debug(f'djmgr - Bot was asked to find a song using : [{url}]')
 
         # Need to extract the video id to get the info from Google API as it does not take a full URL
-        video_id = re.search('.*(?:youtu.be\/|v\/|u\/\w\/|embed\/|watch\?v=)([^#\&\?]*).*', url).group(1)
+        video_id = re.search('.*(?:youtu.be/|v/|u/\w/|embed/|watch\?v=)([^#&?]*).*', url).group(1)
```
Comment (benduhoux) [inline dj_mgr.py:112]: supposedly unnecessary escaping of characters in the regex, we'll see about that...

**Review by Ottawagunner — APPROVED** [review]
aprob

**Pull request merged**